Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I thought you endorsed that cop, though?

http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/FrozenStringLiteralComment